### PR TITLE
Simplified regex in TextTransform url parsing code

### PR DESCRIPTION
**Pull request by hasankhan**

### CODE REVIEW

**Comment by hasankhan** [comment]
Also solves #524

File: JabbR/Infrastructure/TextTransform.cs [hunk 1]
```diff
@@ -42,21 +42,21 @@ private string ConvertTextWithNewLines(string message)
...
+            message = urlPattern.Replace(message, m =>
             {
-
-                string httpPortion = String.Empty;
-                if (!m.Value.Contains("://"))
+                string url = m.Value;
+                if (!url.Contains("://"))
                 {
-                    httpPortion = "http://";
+                    url = "http://" + url;
```
Comment (davidfowl) [inline JabbR/Infrastructure/TextTransform.cs:55]: This looks wrong. What about https and ftp etc? We need better unit tests.

Comment (hasankhan) [inline JabbR/Infrastructure/TextTransform.cs:55]: This is saying if url does not have scheme, prefix it with http. Are you sure its a problem?

Comment (samandmoore) [inline JabbR/Infrastructure/TextTransform.cs:55]: consensus, this looks good.

Comment (davidfowl) [inline JabbR/Infrastructure/TextTransform.cs:55]: We should add a test for twitter urls.

Comment (hasankhan) [inline JabbR/Infrastructure/TextTransform.cs:55]: @davidfowl  can you post a sample url?

Comment (samandmoore) [inline JabbR/Infrastructure/TextTransform.cs:55]: https://twitter.com/jabbr/status/207283403274452992

Comment (samandmoore) [inline JabbR/Infrastructure/TextTransform.cs:55]: we need to add a test that ensures that this fixes #524, can you add that @hasankhan ?

Comment (hasankhan) [inline JabbR/Infrastructure/TextTransform.cs:55]: seems like its already done

**Pull request merged**